metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/File.java

```diff
@@ -86,7 +86,31 @@
     @Column(nullable = false)
     private Type type;
 
-    File() {}
+    /**
+     * Release date control: get the *earliest* release date from all studies which link to this file.
+     */
+    @Formula("(SELECT min(study.release_date) FROM file " +
+            "INNER JOIN analysis_files on file.id = analysis_files.files_id " +
+            "INNER JOIN analysis on analysis_files.analysis_id = analysis.id " +
+            "INNER JOIN study on analysis.study_id = study.id " +
+            "WHERE file.id=id)")
+    @JsonIgnore
+    private LocalDate releaseDate;
+
+    /**
+     * Get the studyIds.
+     */
+
```

Comment:
Extra empty line is present here; I would also suggest to expand description a bit, e. g., "Get the IDs of the studies which link to this object (used for access control)”, something like that